### PR TITLE
CA-9156: Add ansible test and linting workflows

### PR DESCRIPTION
**Pull request by MrManny**
This PR includes:

* add workflow to execute the `ansible-test` command line tool for **sanity tests**.
* add workflow to execute the `ansible-lint` command line tool to **lint the collection with the production profile**
* initializes the `antsibull-changelog` tool 
* update `README.md` and add `CODE_OF_CONDUCT.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
-# Dynatrace + Red Hat Event Driven Ansible: Auto-Remediation
+# Dynatrace + Red Hat Event Driven Ansible
+
+This collection contains the following to Event-Driven Ansible source plugins:
```
Comment (TannerGabriel) [inline README.md:3]:
```suggestion
This collection contains the following two Event-Driven Ansible source plugins:
```

**Review by TannerGabriel — COMMENTED** [review]
Just a small typo

File: README.md [hunk 2]
```diff
@@ -43,6 +50,31 @@ This Event source plugin from Dynatrace captures all problems from your Dynatrac
           name: dt-update-comments.yml
 ```
 
+## dt_webhook 
+
+The dt_webhook event-source plugin is capable of receiving events from Dynatrace via the "Send event to Event-Driven Ansible" workflow action of the Red Hat Ansible for Workflows integration.
```
Comment (agrimmer) [inline README.md:55]: Here a link to the Dynatrace docu would be helpful.

File: galaxy.yml [hunk 3]
```diff
@@ -16,21 +16,23 @@ readme: README.md
...
 # @nicks:irc/im.site#channel'
 authors:
-- Dynatrace <support@dynatrace.com>
+  - Dynatrace <support@dynatrace.com>
 
 
 ### OPTIONAL but strongly recommended
 # A short summary description of the collection
-description: This Event source plugin from Dynatrace captures all problems from your Dynatrace tenant and in conjunction with Ansible EDA rulebooks helps to enable auto-remediation in your environment.
+description: |
```
Comment (agrimmer) [inline galaxy.yml:24]: Do we use the same galaxy config for both plugins? If so, we need to adjust the description.

File: README.md (hunk 2 above)
Comment (dietmar-91) [reply README.md:55]: I will add the link to the Dynatrace Docs of the Red Hat Ansible for Workflows integration

**Review by agrimmer — APPROVED** [review]
LGTM

**Pull request merged**